includes/admin/settings/class-settings-advanced.php

```diff
@@ -153,6 +153,33 @@ public function get_settings() {
 								'disabled' => __( 'Disabled', 'give' ),
 							],
 						],
+						[
+							'name'    => __( 'Setup Page', 'give' ),
+							/* translators: %s: about page URL */
+							'desc'    => sprintf(
+								wp_kses(
+									__( 'Disable this option if you would like to disable the <a href="%s" target="_blank">GiveWP Setup page</a> that displays when GiveWP is first installed.', 'give' ),
+									[
+										'a' => [
+											'href'   => [],
+											'target' => [],
+										],
+									]
+								),
+								esc_url( admin_url( 'edit.php?post_type=give_forms&page=give-setup' ) )
+							),
+							'id'      => 'setup_page_enabled',
+							'type'    => 'radio_inline',
+							'default' => give_is_setting_enabled(
+								\Give\Onboarding\Onboarding::getSetupPageEnabledOrDisabled()
```

Comment:
Please import classes (i.e. `use \Give\Onboarding\Onboarding`) and use just the class name in the code.

Reply:
fixed:https://github.com/impress-org/givewp/compare/e2f41a1ffdd9...172f973f888b